src/UglyToad.PdfPig.Fonts/Type1/Type1FontProgram.cs

```diff
@@ -67,11 +68,47 @@ public class Type1Font
                 return null;
             }
 
-            var bbox = glyph.GetBoundingRectangle();
+            var bbox = GetBoundingRectangle(glyph);
 
             return bbox;
         }
 
+        /// <summary>
+        /// Gets a <see cref="PdfRectangle"/> which entirely contains the geometry of the defined path.
+        /// </summary>
+        /// <returns>For paths which don't define any geometry this returns <see langword="null"/>.</returns>
+        public PdfRectangle? GetBoundingRectangle(List<PdfSubpath> path)
+        {
+            if (path.Count == 0)
+            {
+                return null;
+            }
+
+            var bboxes = path.Select(x => x.GetBoundingRectangle()).Where(x => x.HasValue).Select(x => x.Value).ToList();
+            if (bboxes.Count == 0)
+            {
+                return null;
+            }
+
+            var minX = bboxes.Min(x => x.Left);
+            var minY = bboxes.Min(x => x.Bottom);
+            var maxX = bboxes.Max(x => x.Right);
+            var maxY = bboxes.Max(x => x.Top);
+            return new PdfRectangle(minX, minY, maxX, maxY);
+        }
+
+        /// <summary>
+        /// Get the pdfpath for the character with the given name.
+        /// </summary>
+        public List<PdfSubpath> GetCharacterPath(string characterName)
```

Comment:
As with the other comment I think we should stick to `IReadOnlyList` for these return types to make it clear we're not intending to share data, rather copies of data.

Reply:
makes sense